src/Router.tsx

```diff
@@ -0,0 +1,40 @@
+﻿import React from "react";
+// @ts-ignore
```

Comment:
As discussed - looks like we haven't got the types working properly! I think we shouldn't need this `ts-ignore`

Reply:
deleted ts-ignore